test/ContactBundle.test.ts

```diff
@@ -0,0 +1,18 @@
+import * as assert from 'assert'
+import ContactBundle from '../src/ContactBundle'
+import * as ethers from 'ethers'
+import { PrivateKeyBundle } from '../src'
+
+describe('ContactBundles', function () {
+  it('roundtrip', async function () {
+    const priv = PrivateKeyBundle.generate()
+
+    const cb1 = new ContactBundle((await priv).getPublicKeyBundle())
```

Comment:
Just so that I understand what's going on here, what does the `(await priv).` bit mean? Is it the same as putting the await on line 8?

Reply:
Yes. Moved it up a line to be more standard